### PR TITLE
Dont show unexpected errors in the sourcemap visualizer. fixes #5447

### LINKED ISSUE #5447: When sourcemap tool runs out, our "ready when you are" page looks wrong
image

### PR DESCRIPTION
**Pull request by jasonLaster**
Small fix to avoid showing error modals from unexpected errors in the visualizer. Fix #5447.

### CODE REVIEW

File: src/ui/actions/session.ts [hunk 1]
```diff
@@ -265,13 +265,15 @@ export function clearTrialExpired(): UIThunkAction {
 
 export function setUnexpectedError(error: UnexpectedError, skipTelemetry = false): UIThunkAction {
   return ({ getState, dispatch }) => {
-    const state = getState();
+    if (selectors.getAppMode(getState()) === "sourcemap-visualizer") {
```
[the remark this answers is not in the document]
Comment (jasonLaster) [reply src/ui/actions/session.ts:268]: lol - fair point

**Comment by jaril** [comment]
@jasonLaster rebased and fixed this

**Comment by jasonLaster** [comment]
@jaril looks good

Kicking off a new run because the e2e tests failed with a CodeMirror issue
https://app.replay.io/recording/75de8fd8-04d5-4ea2-a177-765aec0e3f49?point=18497557560099076047842319288763635&time=7728&hasFrames=true

**Comment by jaril** [comment]
Continued in https://github.com/RecordReplay/devtools/pull/6223

**Pull request closed without merging**